examples/nextjs-auth/pages/api/graphql.ts

```diff
@@ -1,10 +1,17 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
 import { createServer } from '@graphql-yoga/node'
-import { NextApiRequest, NextApiResponse } from 'next'
-import { Session } from 'next-auth'
+import type { NextApiRequest, NextApiResponse } from 'next'
+import type { Session } from 'next-auth'
 import { getSession } from 'next-auth/react'
 
-const server = createServer<
+export const config = {
+  api: {
+    // Disable body parsing (required for file uploads)
+    bodyParser: false,
+  },
+}
+
+export default createServer<
```

Comment:
Removing `server` variable makes it look even simpler. I like it

My only concern is `config`. Do you think we should mention it in the `Next.js` section like `Enabling File Uploads`

Reply:
I think it is okay to just make this the default Next.js setup configuration. I don't think we have to explicitly metnion it in the uploads section.

Reply:
For example we don't have this `contentTypeParser` by default
https://www.graphql-yoga.com/docs/integrations/integration-with-fastify
Or this configuration;
https://www.graphql-yoga.com/docs/integrations/integration-with-cloudflare-workers#enabling-subscriptions

I am fine with having this by default but if we are going to mention in the docs. It should be under the platform's dedicated page instead of general `File Uploads` section.

Reply:
My bad I missed it 🤦